dkc/core/models/folder.py

```diff
@@ -62,16 +62,17 @@ class Meta:
     # # Prevent deletion of quotas while a folder references them
     # quota = models.ForeignKey(Quota, on_delete=models.PROTECT)
 
+    @property
     def path_to_root(self) -> List[Folder]:
```

Comment:
Assuming the iteration reversal is dropped, this returns a `RawQuerySet[Folder]`, which is actually ideal, since it's slightly more lazy if we need it to be.